golang/cosmos/app/upgrade.go

```diff
@@ -18,6 +18,7 @@ var upgradeNamesOfThisVersion = []string{
 	"agoric-upgrade-18-devnet",
 	"agoric-upgrade-18-emerynet",
 	"agoric-upgrade-18-basic",
+	"agoric-upgrade-18-basic-2",
```

Comment:
While it doesn't change the behavior, we should have this new name return false in `isPrimaryUpgradeName`

Reply:
I'm also wondering if we should bikeshed the name as it's not really like basic or meant to be applied on its own

Reply:
>I'm also wondering if we should bikeshed the name as it's not really like basic or meant to be applied on its own
what do you suggest?

Reply:
`agoric-upgrade-18-reapply` ? But honestly we can keep as is too.